src/modules/addTodo.js

```diff
@@ -0,0 +1,14 @@
+export default (todoInput, addLocal, tasks) => {
+  if (todoInput.value !== '') {
+    const todo = {
+      id: Date.now(),
+      name: todoInput.value,
+      completed: false,
+    };
```

Comment:
- [ ] Each task object should contain index: [number].
- [ ] index value should start from 1 and last item's index should equal the array length
- [ ] On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.